backend/cmm_helpers.ml

```diff
@@ -3742,10 +3752,10 @@ let allocate_unboxed_int32_array ~elements (mode : Lambda.alloc_mode) dbg =
   in
   let custom_ops =
     (* For odd-length unboxed int32 arrays there are 32 bits spare at the end of
-       the block *)
-    if even_num_of_elts
-    then custom_ops_unboxed_int32_even_array
-    else custom_ops_unboxed_int32_odd_array
+       the block, which are never read. *)
```

Comment:
A custom comparison function (or implementation in the polymorphic compare code, which is currently disabled for these arrays) would already be needed, so I don't think so.

Reply:
it might deserve a comment, probably next/inside the C function that currently implements hashing and comparisons for unboxed int arrays, stating that if/when that function becomes a proper implementation (instead of failing immediately), then code generation should be adapted

Reply:
OK I've added a comment in the C code.